test/exec.node.js

```diff
@@ -81,8 +113,7 @@ describe('exec', () => {
     const args = hang.concat(tok)
 
     const p = exec(args[0], args.slice(1), {}, (err) => {
-      // `tail -f /dev/null somerandom` errors out
```

Comment:
I'm not sure this is correct - on Mac OS at least it complains that `somerandom` doesn't exist but then continues running tailing `/dev/null` and doesn't error out.

Reply:
@achingbrain I didn't understand this part. 

As far as I understood, on my Mac OS it seems ok, as well as in the CI for macos.

Reply:
The comment says the command 'errors out', which I took to mean it exits with a non-zero error code.  When I run it, it prints out a message that `somerandom` doesn't exist but continues running.

Reply:
So, `tail -f` outputs appended data as a file grows. In this case, the file does not exist, but the process still hanging.

I tried to add some data to the file, which I tried to tail and nothing happened as the file does not exist when the command was executed.

This way, I think we can remove the comment as you did!